### PR TITLE
Adds Readme Update to Release Script

### PR DESCRIPTION
**Pull request by danehans**
Updates the release script to update versioned links in `README.md`.

Signed-off-by: Daneyon Hansen <daneyonhansen@gmail.com>

### CODE REVIEW

**Review by stevesloka — CHANGES_REQUESTED** [review]
For me when testing, the `$NEWVERS` doesn't get applied correctly. Example:

```
-$ kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/main/examples/operator/operator.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.99.99/examples/operator/operator.yaml

```

File: hack/release/make-release-tag.sh [hunk 1]
```diff
@@ -5,6 +5,12 @@
...
 # from the previous version. Note: The Envoy image is not managed by this script.
 
+readonly CONTOUR_IMG="docker.io/projectcontour/contour:$NEWVERS"
+readonly OPERATOR_IMG="docker.io/projectcontour/contour-operator:$NEWVERS"
+readonly OPERATOR_EXAMPLE="https://raw.githubusercontent.com/projectcontour/contour-operator/$NEWVERS/examples/operator/operator.yaml"
+readonly CONTOUR_EXAMPLE="https://raw.githubusercontent.com/projectcontour/contour-operator/$NEWVERS/examples/contour/contour.yaml"
+readonly TEST_EXAMPLE="https://projectcontour.io/docs/$NEWVERS/deploy-options/#test-with-ingress"
+readonly CONDUCT_EXAMPLE="https://github.com/projectcontour/contour/blob/$NEWVERS/CODE_OF_CONDUCT.md"
 readonly PROGNAME=$(basename "$0")
 readonly OLDVERS="$1"
```
Comment (stevesloka) [inline hack/release/make-release-tag.sh:15]: Move this up before the other variables introduced.

**Review by skriss — CHANGES_REQUESTED** [review]
Looks good apart from @stevesloka's comment about moving `OLDVERS` and `NEWVERS` up to the top of the var list.

**Review by skriss — APPROVED** [review]
LGTM

**Review by stevesloka — APPROVED** [review]

**Pull request merged**